src/olympia/devhub/tests/test_views.py

```diff
@@ -3835,3 +3835,29 @@ def test_devhub_version_edit_page(self):
     def test_devhub_version_list_page(self):
         url = reverse('devhub.addons.versions', args=[self.addon.slug])
         self.assertNameAndNoXSS(url)
+
+
+class TestSwitchToUnlisted(UploadTest):
```

Comment:
Ugh, not sure why add-on status changes are in that other file. As the filename says, it should be about _versions_ only. Also, `test_views.py` has tests for deleting an add-on.